P3481.bs

```diff
@@ -389,7 +356,7 @@ Checking the cancellation token every iteration may be expensive for certain cas
 A cancellation check requires an acquire memory barrier (which may be too much for really small functors), and might prevent vectorization.
 Thus, implementations may want to check the cancellation every *N* iterations; *N* can be a statically known number, or can be dynamically determined.
 
-As `bulk`, `bulk_chunked`, and `bulk_unchunked` are customization points, we need to specify these contraints to all three of them.
+As `bulk`, `bulk_chunked`, and `bulk_unchunked` are customization points, we need to specify these constraints to all three of them.
 
 
 ## Exception handling ## {#design.exceptions}
```

Comment:
I don't think the option 2 for exceptions is written accurately. If you have any ideas for improvement you are welcome to apply them. Otherwise, let's discuss it when we have a chance.

Reply:
Adding reference to p0333r0 to clarify. See also https://github.com/cplusplus/sender-receiver/issues/276#issuecomment-2225083545.

Reply:
Still "reducing exceptions" and "exception list" seem like two different approaches to me. I don't want to change that at the last moment, but perhaps we might want to discuss that to no argue with each other during C++ committee session